tests/integration/tests/test_cleanup.py

```diff
@@ -64,15 +64,28 @@ def test_node_cleanup_new_containerd_path(instances: List[harness.Instance]):
 
     boostrap_config = yaml.safe_load(containerd_path_bootstrap_config)
     new_containerd_paths = [
-        os.path.join(boostrap_config["containerd-base-dir"], "k8s-containerd", p)
+        os.path.join(
+            boostrap_config["containerd-base-dir"], "k8s-containerd", p.lstrip("/")
+        )
         for p in CONTAINERD_PATHS
     ]
+
+    # We expect containerd to use our custom paths instead of the default ones.
+    # However, the test fixture places registry configuration in /etc/containerd
```

Comment:
Can we please fix this in the fixture instead of cleaning up the paths after the fact?

Reply:
Thanks for the suggestion. I think we can add a marker for custom containerd paths to accommodate this test.